babel.config.js

```diff
@@ -39,67 +21,9 @@ const config = {
 			),
 			{ async: isCalypsoClient && codeSplit },
 		],
-		'@babel/plugin-proposal-class-properties',
-		'@babel/plugin-proposal-export-default-from',
-		'@babel/plugin-proposal-export-namespace-from',
-		'@babel/plugin-syntax-dynamic-import',
-		[
-			'@babel/transform-runtime',
-			{
-				corejs: false, // we polyfill so we don't need core-js
-				helpers: true,
-				regenerator: false,
-				useESModules: false,
-			},
-		],
 		isCalypsoClient && './inline-imports.js',
 	] ),
-	overrides: [
-		{
-			test: [ './client/gutenberg/extensions', './packages/jetpack-blocks' ],
-			plugins: [
-				[
-					'@wordpress/import-jsx-pragma',
-					{
-						scopeVariable: 'createElement',
-						source: '@wordpress/element',
-						isDefault: false,
-					},
-				],
-				[
-					'@babel/transform-react-jsx',
-					{
-						pragma: 'createElement',
-					},
-				],
-			],
-		},
-	],
 	env: {
-		build_pot: {
```

Comment:
are we going to be using this `build_pot` in Jetpack?

Reply:
I don't think we need it — if we push block JavaScript sources to SVN, WordPress.org's Glotpress process will pick up any `__()` functions from our files and generate pot files automatically.

In fact, I don't think Jetpack generates pot files even for PHP files manually, it's all automatic these days?

cc @zinigor

Reply:
Yep, that was my thinking!